app/views/groups/members/_member.html.erb

```diff
@@ -53,22 +55,59 @@
     <div>
       <% if member.user.email != member.created_by.email %>
         <%= viral_time_ago(original_time: member.created_at) %>
-        by <span class="block">
-        <%= member.created_by.email %>
+        by
+        <span class="block">
+          <%= member.created_by.email %>
         </span>
       <% else %>
         <%= viral_time_ago(original_time: member.created_at) %>
       <% end %>
 
       <%= turbo_frame_tag("member-#{member.id}-access-level-updated") do %>
         <% if member.created_at < member.updated_at %>
-          <%= render UpdatedComponent.new(
-            updated_at: member.updated_at
-          ) %>
+          <%= render UpdatedComponent.new(updated_at: member.updated_at) %>
         <% end %>
       <% end %>
     </div>
   </td>
+  <td class="px-6 py-4">
+    <% if (current_user != member.user && allowed_to?(:update_member?, @namespace)) && !member_source.key?(:inherited_namespace_path) %>
```

Comment:
Logged in as a user with `Maintainer` access level I can update the expiration for an `Owner`. We need to add an `&&` to the first part of the if statement. Take a look at the `access_level` column in this file which has the missing `&&`

![image](https://github.com/phac-nml/irida-next/assets/25466211/67f05311-7210-4594-adec-6ef5bf96836d)

Reply:
a10c35a94bb03411803d11118e64ef39bdfb6913